components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS2Parser.java

```diff
@@ -1276,6 +1276,7 @@ private boolean isDefaultGiven(String swaggerContent) throws APIManagementExcept
     public String processOtherSchemeScopes(String swaggerContent) throws APIManagementException {
         if (!isDefaultGiven(swaggerContent)) {
             Swagger swagger = getSwagger(swaggerContent);
+            swagger = injectMgwThrottlingExtensionsToDefault(swagger);
```

Comment:
Shall we move this to https://github.com/wso2/carbon-apimgt/blob/62dbf8e46ccaa067d8bf5c1f4ed028502fd2111f/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java#L1325-L1329